perfkitbenchmarker/providers/gcp/gcp_managed_relational_db.py

```diff
@@ -64,6 +64,7 @@ class GCPManagedRelationalDb(managed_relational_db.BaseManagedRelationalDb):
   # These are the constants that should be specified in GCP's cloud SQL command.
   GCP_PRICING_PLAN = 'PACKAGE'
   MYSQL_DEFAULT_PORT = 3306
+  VALID_POSTGRES_MACHINE_TYPES = ['db-f1-micro', 'db-g1-small']
```

Comment:
GCP support a lot more valid postgres machine types. why are these two listed as the only valid ones here?

Actually GCP also supports custom machien types for postgres, we can add that later but make sure you add a todo item here.

(the best way to get up to speed with GCP postgres support is to try that out manually first).